test/react16_suggestors/react_style_maps_updater_test.dart

```diff
@@ -213,8 +213,8 @@ main() {
             main() {
               Foo()
               ${manualVariableCheckComment(keysOfModdedValues: [
-                'width',
-              ])}
+            'width',
```

Comment:
I was running the formatter on 2.13.4, and this changed 🤷‍♂️  If there's a different version of the formatter I should be running (or that sounds weird that it changed) let me know!!